### PR TITLE
Update repositories.json

### PR DESCRIPTION
**Pull request by benjaaguirre10**
# Application whitelist

- Whitelist is in a place acting as a first defence against malicious people that want to misuse Flux in order to for example mine cryptocurrencies or distribute illegal or copyrighted material. 
- To whitelist an application for Flux, please adjust helpers/repositories.json file by adding your desired whitelist for a specific docker hub organisation (recommended), docker hub image or target a specific tag of an image:
- Valid Examples: runonflux, runonflux/website, runonflux/website:latest

# What do you want to Run On Flux?

Run a vpn on flux with the ssl vpn of forticlient

# Is your desired application running somewhere already?

*Please provide a POC link to application if it is already running somewhere. (optional)*

# Is your application open source?

*Please provide a source code (optional)*

# Checklist:

- [x] Whitelist of application is only modifying repositories.json file
- [ ] repositories.json is still a valid JSON file
- [ ] Only whitelists single docker hub username (one whitelist at a time, more whitelists, more PRs)
- [x] No other whitelist has been deleted
- [x] I agree with ToS https://cdn.runonflux.io/Flux_Terms_of_Service.pdf
- [x] Application follows ToS - Application is not malicious. Application is not a scam. Application does what is meant to do and does not mislead in any way. Application does not do anything illegal. Application is not a mining application (not even bandwidth mining).
- [x] In case application receives multiple reports, behaves maliciously, it will be blacklisted and removed from the network.

### CODE REVIEW

File: helpers/repositories.json [hunk 1]
```diff
@@ -283,4 +283,7 @@
    "pgerringer",
    "sysadminblaze",
    "fluxdapp"
```
Comment (Cabecinha84) [inline helpers/repositories.json:285]: missing , after the  "fluxdapp"

File: helpers/repositories.json [hunk 2]
```diff
@@ -283,4 +283,7 @@
...
+   "ncleguizamon/forticlient:latest"
```
Comment (Cabecinha84) [inline helpers/repositories.json:286]: missing , after the "ncleguizamon/forticlient:latest"

File: helpers/repositories.json [hunk 3]
```diff
@@ -283,4 +283,7 @@
...
+   "utunnelio/utunnel:production-18.04"
+
```
Comment (Cabecinha84) [inline helpers/repositories.json:288]: please remove the empty extra line

**Comment by TheTrunk** [comment]
Hi, please follow the pull request form

**Pull request closed without merging**